docs/release_notes.rst

```diff
@@ -19,7 +19,7 @@ This release introduces the following:
 - BIG-IP® vCMP® compatibility
 - Bug fixes
 
-See the `changelog <https://github.com/F5Networks/f5-openstack-lbaasv2-driver/compare/v9.0.2...v9.0.3>`_ for the full list of changes in this release.
+See the `changelog <https://github.com/F5Networks/f5-openstack-lbaasv2-driver/compare/v9.0.3...v9.1.0>`_ for the full list of changes in this release.
```

Comment:
Delete line 29 (should happen via merge from liberty), but keep lines 30-31 since this is Mitaka-specific functionality that we will add.